lawluigi_configs/KingMaker_luigi.cfg

```diff
@@ -14,7 +14,7 @@ ENV_NAME = KingMaker
 
 ; grid storage protocol and path usable from submitting machine and worker nodes of cluster
 ; job in- and output will be stored in $wlcg_path under subdirectory of analysis $name
-wlcg_path = srm://cmssrm-kit.gridka.de:8443/srm/managerv2?SFN=/pnfs/gridka.de/cms/disk-only/store/user/${USER}/CROWN_samples
+wlcg_path = root://cmsxrootd-kit.gridka.de//store/user/${USER}/CROWN_samples
```

Comment:
xrootd-paths appear to only work when the environment is sourced via cvmfs. If the environment is not present, there the copying of the tarballs via gfal-copy fails with `Missing source`. Replacing gfal-copy with xrdcp worked for me.

Reply:
you mean these here: https://github.com/KIT-CMS/KingMaker/blob/2ad80971865329f9333d4216f585b8de11e10619/processor/setup_law_remote.sh#L43 ?

Reply:
Yes. In addition, I think that there is an issue with caching. When I change something in the process tar, the new version is uploaded correctly, but if I use xrdcp to download it in the setup of the job a cached version is used instead.

Reply:
yes, that is propably the dcache thing, where replaceing files is not an expected thing to do ..

Reply:
we could add something like a random string to the end of the tarball files to make sure, that a new file is written everytime ?